grunt/tasks/scss.js

```diff
@@ -14,6 +14,8 @@ module.exports = {
         files: [{
           expand: true,
           src: [
+            'node_modules/cartoassets/src/scss/utilities/*.scss',
```

Comment:
I wouldn't use wildcards, but explicitely list the files, even better: have an entry file